src/components/SwapForm/hooks/useCheckStablePairSwap.ts

```diff
@@ -0,0 +1,22 @@
+import { Currency } from '@kyberswap/ks-sdk-core'
+
+import { STABLE_COINS_ADDRESS } from 'constants/tokens'
+import { useActiveWeb3React } from 'hooks'
+
+const useCheckStablePairSwap = (currencyIn: Currency | undefined, currencyOut: Currency | undefined) => {
+  const { chainId } = useActiveWeb3React()
+
+  const isStablePairSwap = Boolean(
+    chainId &&
+      currencyIn &&
+      currencyOut &&
+      currencyIn.wrapped.address &&
```

Comment:
can we remove `currencyIn.wrapped.address &&
      currencyOut.wrapped.address &&` ?

Reply:
OK. removed